go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -761,12 +542,12 @@ func (qre *QueryExecutor) getStreamConn() (*connpool.DBConn, error) {
 }
 
 func (qre *QueryExecutor) qFetch(logStats *tabletenv.LogStats, parsedQuery *sqlparser.ParsedQuery, bindVars map[string]*querypb.BindVariable) (*sqltypes.Result, error) {
-	sql, sqlWithoutComments, err := qre.generateFinalSQL(parsedQuery, bindVars, nil, "")
+	sql, sqlWithoutComments, err := qre.generateFinalSQL(parsedQuery, bindVars)
 	if err != nil {
 		return nil, err
 	}
 	// Check tablet type.
-	if qre.tsv.qe.enableConsolidator || (qre.tsv.qe.enableConsolidatorReplicas && qre.tabletType != topodata.TabletType_MASTER) {
+	if qre.tsv.qe.enableConsolidator || (qre.tsv.qe.enableConsolidatorReplicas && qre.tabletType != topodatapb.TabletType_MASTER) {
 		q, original := qre.tsv.qe.consolidator.Create(string(sqlWithoutComments))
```

Comment:
consolidator is default to true and hence test does not go in else part. Can a test be added?

Reply:
This test requires creating a delay in query execution, which the test framework can't emulate. So, it's covered in the `go/vt/vttablet/endtoend/misc_test.go`.